cmd/kvload/main.go

```diff
@@ -2,36 +2,28 @@ package kvload
 
 import (
 	"fmt"
+	"strings"
+	"sync"
+	"time"
 
-	"github.com/bmeg/golib"
 	"github.com/bmeg/grip/gripql"
 	"github.com/bmeg/grip/kvgraph"
 	"github.com/bmeg/grip/kvi"
+	"github.com/bmeg/grip/log"
 	"github.com/bmeg/grip/util"
-	log "github.com/sirupsen/logrus"
+	"github.com/paulbellamy/ratecounter"
+	"github.com/sirupsen/logrus"
```

Comment:
It's a little weird that `github.com/bmeg/grip/log` has been added, and wraps almost all of the logger API, but the config flags still need to be pulled from logrus